eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -36,6 +36,11 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
This will only check the links under the service directory. In the other languages there is an "aggregate-reports" pipeline which we used to check links in the other MD files like in the root of the repo. @mitchdenny @bsiegel is there an aggregate reports type pipeline for this repo yet?